web/templates/modal_banner/unsubscribed_participants_warning_banner.hbs

```diff
@@ -0,0 +1,22 @@
+{{#> modal_banner . }}
+    <p class="banner_message">
+        {{#if few_unsubscribed_participants}}
+            {{#tr}}Some topic participants are not subscribed to{{/tr}}<br/>
+            <span class="stream-privacy-modified-color-{{stream_id}} stream-privacy filter-icon" style="color: {{stream_color}}">
+                {{> ../stream_privacy . }}
+                {{stream_name}}:&nbsp;
+            </span>
+            {{#each unsubscribed_participants}}
+                <strong>{{full_name}}</strong>{{#unless (eq @index ../last_index)}},&nbsp;{{else}}.{{/unless}}
+            {{/each}}
```

Comment:
I assume you're working on replacing this with the `ListFormat` stuff.

Reply:
It was addressed, the variable `unsubscribed_participantn_names_list` which is constructed via `format_array_as_list_with_conjuction` is now passed